src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java

```diff
@@ -40,6 +40,9 @@ public class ObTableClientQueryImpl extends AbstractTableQueryImpl {
 
     private Row                 rowKey;       // only used by BatchOperation
 
+    public void AddAggregation(AggregationType aggType, String aggColumn) {
```

Comment:
Shall we add some test cases for your new feature?

Reply:
add some test cases